frontend/src/components/MatchingPairs/MatchingPairs.tsx

```diff
@@ -152,6 +154,7 @@ const MatchingPairs = ({
     };
 
     const finishTurn = () => {
+        setStartOfTurn(performance.now());
```

Comment:
Why set this here and at line 145? Wouldn't it be enough to do it in one of those two places?